opm/common/OpmLog/LogBackend.hpp

```diff
@@ -75,6 +79,7 @@ namespace Opm
         int64_t m_mask;
         std::shared_ptr<MessageFormatterInterface> m_formatter;
         std::shared_ptr<MessageLimiter> m_limiter;
+        int64_t m_level;
```

Comment:
> Then we could use m_mask 

Yes - in my opinion that system is already in place; but of course - it can certainly be improved. I suggest you look at the current `m_mask` based solution and see how/if you think that should be improved.

If nothing else I certainly agree with: 

> @jokva: please use an unsigned type. uint64_t's likely to be the one.